context/app/static/js/components/entity-search/sidebar/ConfigureSearchTable/ConfigureSearchTable.jsx

```diff
@@ -26,21 +27,23 @@ function ConfigureSearchTable({
     setSearchBarFieldName(event.target.value);
   };
 
+  const filteredFields = filterFieldConfigs(getFieldEntriesSortedByConfigureGroup(availableFields), searchBarFieldName);
+
   return (
     <FlexGrow>
       <StyledSearchBar fullWidth value={searchBarFieldName} onChange={handleChange} />
-      <TableContainer component={Paper}>
-        <Table>
-          <TableHead>
-            <TableRow>
-              <TableCell />
-              <TableCell>Add Facet</TableCell>
-              <TableCell>Add Column</TableCell>
-            </TableRow>
-          </TableHead>
-          <TableBody>
-            {filterFieldConfigs(getFieldEntriesSortedByConfigureGroup(availableFields), searchBarFieldName).map(
-              ([fieldName, fieldConfig]) => (
+      {filteredFields.length > 0 ? (
```

Comment:
Slight preference for bringing the shorter side of an if-then to the top, for readability. Another option might be factoring out everything under `<TableContainer>` as a separate component.